downstream/assemblies/eda/assembly-simplified-event-routing.adoc

```diff
@@ -0,0 +1,27 @@
+
+[id="simplified-event-routing"]
+
+= Simplfied event routing
+
+Simplified event routing enables {EDAcontroller} to capture and analyze data from various remote systems using event streams. Event streams allow you to send events from a remote system like GitHub or GitLab into Event-Driven Ansible controller. You can attach 1 or more event streams to an activation by swapping out sources in a rulebook. 
```

Comment:
Missed attribute for Event-Driven Ansible controller in the second sentence here.

Reply:
Done.